Sources/Kitura/RouterRequest.swift

```diff
@@ -184,6 +184,21 @@ public class RouterRequest {
         _ = try serverRequest.read(into: &data)
         return try JSONDecoder().decode(type, from: data)
     }
+    
+    /// Read the URLEncoded body of the request as a Codable object.
+    ///
+    /// - Parameter type: Codable object to which the body of the request will be converted.
+    /// - Throws: Socket.Error if an error occurred while reading from a socket.
+    /// - Throws: `DecodingError.dataCorrupted` if values requested from the payload are corrupted, or if the given data is not valid JSON.
+    /// - Throws: An error if any value throws an error during decoding.
+    /// - Returns: The instantiated Codable object
+    public func readURLEncoded<T: Decodable>(as type: T.Type) throws -> T {
+        let bodyAsString = try self.readString()
```

Comment:
I think this could just be `body` not `bodyAsString`. The use of `readString()` hints enough that it's a String.